package.json

```diff
@@ -1,7 +1,5 @@
 {
-    "name": "containerar",
-    "widgetName": "ContainerAR",
-    "version": "1.0.0",
+    "name": "ar-widgets-native",
     "description": "The container used to start an AR experience",
     "copyright": "© Mendix Technology BV 2022. All rights reserved.",
```

Comment:
Probably year also need an update (I have to do same for `mendix/web-widgets`

Reply:
2022 -> 2023

Reply:
Oh good catch, totally forgot about that